README.md

```diff
@@ -6,17 +6,15 @@ the release program.*  To build your own release see [CONTRIBUTING.md](CONTRIBUT
 
 ## Releases
 
-### 1.9 Kubernetes
+### 1.10 Kubernetes
 
-Release:
+[Release](https://github.com/kubernetes-sigs/kubebuilder/releases):
 
-- [v1beta1.1](https://github.com/kubernetes-sigs/kubebuilder/releases/tag/v1beta1.1)
+- [v0.1.7](https://github.com/kubernetes-sigs/kubebuilder/releases/tag/v0.1.7)
 
```

Comment:
We should probably replace this with `https://github.com/kubernetes-sigs/kubebuilder/releases/latest` which always point to latest release. Want to avoid updating README for every release.

Reply:
done